Watchman.Discord/Areas/Responses/Services/ResponsesMessageService.cs

```diff
@@ -64,20 +64,17 @@ public async Task PrintResponses(string commandArgument, Contexts contexts)
 
         private string GetResponseWithVariableList(Response response)
         {
-            var stringBuilder = new StringBuilder("\n__Dostępne zmienne:__");
+            var result = "\n__Dostępne zmienne:__";
             if (response.AvailableVariables.Any())
             {
-                foreach (var variable in response.AvailableVariables)
-                {
-                    stringBuilder.Append($" `{variable}`");
-                }
+                result += response.AvailableVariables.Select(s => $" `{s}`").Aggregate((a, b) => a+b);
             }
             else
             {
-                stringBuilder.Append(" brak");
+                result += " brak";
             }
 
-            return GetRawMessage(response.Message) + stringBuilder;
+            return GetRawMessage(response.Message) + result;
```

Comment:
this.GetRawMessage

Reply:
właśnie widziałem że tak piszecie zawsze z this. ale mi Rider wywala zawsze ("useless .this"). Już naprawiam.

Reply:
zmień w ustawieniach żeby wymagał this.

Reply:
pewnie w ustawieniach ridera masz to konfigurowalne

Reply:
nie da się ustawić żeby wymagało, ale da się ustawić żeby się linter na mnie nie wydzierał